data/json/recipes/recipe_others.json

```
@@ -3778,12 +3778,12 @@
     "skill_used": "survival",
     "skills_required": [ "cooking", 1 ],
     "difficulty": 1,
-    "time": "30 m",
+    "time": "5 m",
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [
       [ [ "salt_water", 2 ], [ "saline", 10 ], [ "salt", 10 ] ],
```

Salt cost should be reduced as well, so it doesn't cost 6x the amount of salt to cure the same amount of hide (as you did with pelts).
```suggestion
       [ [ "salt_water", 1 ], [ "saline", 2 ], [ "salt", 2 ] ],
```

---

I adjusted the cost for the fur, but forgot it here. Thanks!